variables.tf

```diff
@@ -1,24 +1,276 @@
+# PowerVS NetWeaver Instance parameters
+# Copyright 2022 IBM
+#####################################################
+
+variable "pvs_netweaver_instance_name" {
+  description = "Name of instance which will be created"
+  type        = string
+}
+
+variable "pvs_netweaver_image_name" {
+  description = "Image Names to import into the service"
+  type        = string
+}
+
+variable "pvs_netweaver_number_of_instances" {
+  description = "Number of instances"
+  type        = string
+  default     = 1
+}
+
+variable "pvs_netweaver_server_type" {
+  description = "Processor type e980, s922, s1022 or e1080"
+  type        = string
+  default     = "s922"
+}
+
+variable "pvs_netweaver_cpu_proc_type" {
+  description = "Dedicated or shared processors"
+  type        = string
+  default     = "shared"
+}
+
+variable "pvs_netweaver_number_of_processors" {
+  description = "Number of processors"
+  type        = string
+}
+
+variable "pvs_netweaver_memory_size" {
+  description = "Amount of memory"
+  type        = string
+}
+
+variable "pvs_netweaver_storage_config" {
+  description = "DISKS To be created and attached to node.Comma separated values"
+  type        = map(any)
+  default = {
+    names      = ""
+    paths      = ""
+    disks_size = ""
+    counts     = ""
+    tiers      = ""
+  }
+}
+
+#####################################################
+# PowerVS Shared FS Instance parameters
+# Copyright 2022 IBM
+#####################################################
+
+variable "pvs_share_instance_name" {
+  description = "Name of instance which will be created"
+  type        = string
+}
+
+variable "pvs_share_image_name" {
+  description = "Image Names to import into the service"
+  type        = string
+}
+
+variable "pvs_share_number_of_instances" {
+  description = "Number of instances"
+  type        = string
+}
+
+variable "pvs_share_server_type" {
+  description = "Processor type e980, s922, s1022 or e1080"
+  type        = string
+  default     = "s922"
+}
+
+variable "pvs_share_cpu_proc_type" {
+  description = "Dedicated or shared processors"
+  type        = string
+  default     = "shared"
+}
+
+variable "pvs_share_number_of_processors" {
+  description = "Number of processors"
+  type        = string
+  default     = 0.5
+}
+
+variable "pvs_share_memory_size" {
+  description = "Amount of memory"
+  type        = string
+  default     = 2
+}
+
+variable "pvs_share_storage_config" {
+  description = "DISKS To be created and attached to node.Comma separated values"
+  type        = map(any)
+  default = {
+    names      = ""
+    paths      = ""
+    disks_size = ""
+    counts     = ""
+    tiers      = ""
+  }
+}
+
+#####################################################
+# PVS SAP instance Initialization
+# Copyright 2022 IBM
+#####################################################
+
+variable "access_host_or_ip" {
+  description = "Public IP of Bastion/jumpserver Host"
+  type        = string
+}
+
+/***
```

Comment:
Remove comments

Reply:
next PRs will introduce the usage. will fix then.